README.md

```diff
@@ -17,24 +17,34 @@ This analysis repository presumes that the following python packages are availab
  * dask
  * bottleneck
  * basemap
- * nco with `conda install -c conda-forge nco`
+ * lxml
+ * nco
 
 You can easily install them via the conda command:
 
 ```
-conda install -c conda-forge numpy scipy matplotlib netCDF4 xarray dask bottleneck basemap nco
+conda config --add channels conda-forge
+conda install numpy scipy matplotlib netCDF4 xarray dask bottleneck basemap lxml nco
```

Comment:
Some updates to the required libraries that I intended to add with parallel tasks but which have languished.